docs/user_roles.md

```diff
@@ -32,13 +32,21 @@ You can do so from the rails console:
 - $ `rails c`
 - > `User.find_by(username: '<your username>').update_attribute(:permissions, 1)`
 
+If you need a higher level of permission above admin. Consider giving your main account super admin permissions instead:
+- $ `rails c`
+- > `User.find_by(username: '<your username>').update_attribute(:permissions, 3)`
+
 ## Instructor role
 
 To use the dashboard as an instructor, you can either use an account with admin
 permissions, or use another account and select the 'instructor' option from the
 onboarding process (http://localhost:3000/onboarding). Either way, you'll be able
 to create new courses/programs after that from the home page.
 
+## Assigning special roles
+
+To assign users to special roles. Go to the `/settings` page in the dashboard (http://localhost:3000/settings). This will allow you to give users admin privileges and other special user permissions such as classroom program manager, technical help staff, Wikipedia expert and many others. Note that your account needs to have super admin permissions to access the page (refer to [Admin permissions](#admin-permissions) for more information) 
```

Comment:
"To assign users to special roles, go"

Also, add a period at the end.

Reply:
Ok, done